demo/src/containers/Chat/Chat.tsx

```diff
@@ -2,16 +2,18 @@ import { useCallback, useEffect, useRef } from 'react';
 import { MessageComponent } from '../../components/MessageComponent';
 import { MessageInput } from '../../components/MessageInput';
 import { useMessages } from '../../hooks/useMessages';
-import { useTypingIndicators } from '../../hooks/useTypingIndicators.ts';
 import { useReactions } from '../../hooks/useReactions';
 import { ReactionInput } from '../../components/ReactionInput';
 import { ConnectionStatusComponent } from '../../components/ConnectionStatusComponent/ConnectionStatusComponent.tsx';
-import { useChatConnection } from '@ably/chat/react';
+import { useChatConnection, useTyping } from '@ably/chat/react';
 import { ConnectionLifecycle } from '@ably/chat';
 
 export const Chat = () => {
   const { loading, clientId, messages, sendMessage } = useMessages();
-  const { startTyping, stopTyping, typers } = useTypingIndicators();
+
+  const { start, stop, currentlyTyping } = useTyping({
+    updateInterval: 1000,
```

Comment:
If we keep the throttling, maybe we should think about the name of this parameter. `maxUpdateInterval`? Also, not a big deal but there's a potential confusion with the `typingTimeout`, the timer that calls `stop()` for you.